packages/contracts-bedrock/snapshots/storageLayout/L1CrossDomainMessenger.json

```diff
@@ -113,17 +106,52 @@
   },
   {
     "bytes": "20",
-    "label": "otherMessenger",
+    "label": "spacer_207_0_20",
     "offset": 0,
     "slot": "207",
-    "type": "contract CrossDomainMessenger"
+    "type": "address"
   },
   {
-    "bytes": "1376",
-    "label": "__gap",
+    "bytes": "20",
+    "label": "xDomainMsgSender",
     "offset": 0,
     "slot": "208",
-    "type": "uint256[43]"
```

Comment:
This gap is now 41 slots long, and is shifted down. 
It allows for the new `xDomainMsgSender` and `initialized` slots to each occupy a full 32 bytes slot, at the beginning of where this gap was.

The key thing to note is that in this layout is that the pre-existing storage slots are either:
1. unaffected
2. have been replaced by a spacer